src/middlewares/likes.middleware.js

```diff
@@ -6,10 +6,6 @@ const util = new Util();
 export default async (req, res, next) => {
   try {
     const user = await models.user.findOne({ where: { email: req.auth.email } });
-    if (!user) {
```

Comment:
why was this part deleted though?

Reply:
> why was this part deleted though?
I removed it because it doesn't have to be there though its a case that to be sure that the user with the token is not yet deleted and for that concern, I think on delete user feature there should be the way to grab all token from deleted user and invalid it anyway not in this feature. but for this feature to be safe let me add codes back.

Reply:
Oh okay.